### PR TITLE
Name to package

### LINKED ISSUE #10: TypeError: 'ocrd_froc.processor' is not a package
`make check`for ocrd_all fails with the error message `TypeError: 'ocrd_froc.processor' is not a package` since at least ocrd_all v2024-02-01.

I get the same error when I just run `ocrd-froc-recognize` without any argument:
```
$ ocrd-froc-recognize
Traceback (most recent call last):
  File "/home/stweil/src/github/OCR-D/ocrd_all_test/venv/bin/ocrd-froc-recognize", line 5, in <module>
    from ocrd_froc.cli.ocrd_cli import cli
  File "/home/stweil/src/github/OCR-D/ocrd_all_test/venv/lib/python3.11/site-packages/ocrd_froc/cli/ocrd_cli.py", line 7, in <module>
    from ..processor import FROCProcessor
  File "/home/stweil/src/github/OCR-D/ocrd_all_test/venv/lib/python3.11/site-packages/ocrd_froc/processor.py", line 24, in <module>
    OCRD_TOOL = loads(resource_string(__name__, 'ocrd-tool.json'))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/stweil/src/github/OCR-D/ocrd_all_test/venv/lib/python3.11/site-packages/ocrd_utils/introspect.py", line 58, in resource_string
    with open(resource_filename(pkg, fname), 'r', encoding='utf-8') as f:
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/stweil/src/github/OCR-D/ocrd_all_test/venv/lib/python3.11/site-packages/ocrd_utils/introspect.py", line 54, in resource_filename
    ref = importlib_resources.files(pkg) / fname
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/resources/_common.py", line 22, in files
    return from_package(get_package(package))
                        ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/resources/_common.py", line 55, in get_package
    raise TypeError(f'{package!r} is not a package')
TypeError: 'ocrd_froc.processor' is not a package
```

### PR DESCRIPTION
**Pull request by kba**
fix #10

### CODE REVIEW

**Review by stweil — APPROVED** [review]
Thank you! This fixes issue #10.

**Comment by stweil** [comment]
See [result of complete test matrix](https://github.com/stweil/ocrd_froc/actions/runs/8692367293) where now all checks pass.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,11 +17,11 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          # - '3.7'
+          - '3.7'
```
Comment (stweil) [inline .github/workflows/ci.yml:20]:
```suggestion
```

Comment (stweil) [reply .github/workflows/ci.yml:20]: What about dropping the unsupported Python 3.7 for all OCR-D code?

**Comment by kba** [comment]
I still get 

> See [result of complete test matrix](https://github.com/stweil/ocrd_froc/actions/runs/8692367293) where now all checks pass.

I still see an issue with 3.9. I think it's best to rely on `importlib_resources` package for `<= 3.9` (instead of just `<= 3.8`) because there the behavior is correct. Will prepare a PR in core for that.

**Comment by stweil** [comment]
Thanks, @kba, for your work. It's nice to have green CI tests now.

**Comment by bertsky** [comment]
> I still see an issue with 3.9. I think it's best to rely on `importlib_resources` package for `<= 3.9` (instead of just `<= 3.8`) because there the behavior is correct.

[Matplotlib would agree](https://github.com/matplotlib/matplotlib/blob/d901275d7cc0607db1dfc1385196daa67e445a47/lib/matplotlib/style/core.py#L21-L26) with you.

**Pull request merged**